airbyte-integrations/connectors/source-chargebee/source_chargebee/components.py

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+from dataclasses import dataclass
+from typing import Optional
+
+from airbyte_cdk.sources.declarative.transformations.transformation import RecordTransformation
+from airbyte_cdk.sources.declarative.types import Config, Record, StreamSlice, StreamState
+
+
+@dataclass
+class CustomFieldTransformation(RecordTransformation):
```

Comment:
could you create an issue for this custom component like described [here](https://docs.google.com/document/d/1LB4gBk0slHj3zNmq1Qtj1U7VkKyTN9eFsVldgQ8T4HQ/edit?usp=drivesdk)?

Reply:
done https://github.com/airbytehq/airbyte/issues/23134